packages/js/product-editor/src/components/index.ts

```diff
@@ -5,3 +5,4 @@ export {
 export { WooProductFieldItem as __experimentalWooProductFieldItem } from './woo-product-field-item';
 export { WooProductSectionItem as __experimentalWooProductSectionItem } from './woo-product-section-item';
 export { WooProductTabItem as __experimentalWooProductTabItem } from './woo-product-tab-item';
+export { Editor } from './editor';
```

Comment:
Should we export the editor components as `__experimental` for now?

In theory I am fine foregoing this, but upon initial release of the `product-editor` package we should either label the package as a whole `experimental` at first (versioning beta releases) and only upon a non-beta we either add on the `__experimental` or make sure things are polished.
This is not a blocker to this PR, but something we should decide upon, as we don't want to mix and match. 

Given most of the other components within this package contain `__experimental` we probably want to do that for this as well.

Reply:
Done -> [b6af5af](https://github.com/woocommerce/woocommerce/pull/37023/commits/b6af5af8cfecac616d612fe8f13eb78ac327d32b)